repo_resource/common.py

```diff
@@ -105,12 +105,24 @@ def getRevision(remote, remoteUrl, project, branch):
             if is_sha1(branch):
                 return (remote + '/' + project, branch)
```

Comment:
nitpick: commit message: Remove "this patch" and just use imperative mood.

Reply:
yes